digital-wallet-widgets/wallettoken.md

```diff
@@ -6,7 +6,7 @@ Generating your wallet token and embedding the wallet button are mandatory steps
 
 To use your Digital Wallets, you must first generate a `wallet_token`. The `wallet_token` forms a part of the request body for your Applepay, GooglePay, FlyPay and Paypal initialisation.
```

Comment:
This is wrong - only after the user has tapped the button will the widget trigger a callback to the Merchant app requesting a wallet token.